src/main/target/SITL/target.c

```diff
@@ -79,6 +85,15 @@ int lockMainPID(void) {
 #define RAD2DEG (180.0 / M_PI)
 #define ACC_SCALE (256 / 9.80665)
 #define GYRO_SCALE (16.4)
+
+//#define SIMULATOR_IP "127.0.0.1"
+#define SIMULATOR_IP "172.21.32.1"
```

Comment:
Currently, the user should modify this IP before testing SITL. I am working on use arguement.